src/screens/ProfileScreen/index.js

```diff
@@ -48,7 +49,6 @@ import {
   updateBioProfile,
   updateImageProfile
 } from '../../service/profile';
-import {deleteAnonymousPost, deletePost, viewTimePost} from '../../service/post';
 import {downVote, upVote} from '../../service/vote';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `deleteAnonymousPost`, `deletePost`, and `viewTimePost` from `../../service/post` has been moved to a different location. This change is fine as long as it doesn't affect the order of execution or dependencies.
```diff
- import {deleteAnonymousPost, deletePost, viewTimePost} from '../../service/post';
+ updateBioProfile,
+ updateImageProfile
+ } from '../../service/profile';
+ import {downVote, upVote} from '../../service/vote';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->